daemon/cmd/hubble.go

```diff
@@ -110,6 +111,25 @@ func (d *Daemon) launchHubble() {
 		}
 	}
 
+	if option.Config.HubbleDropEvents {
+		logger.Infof("Starting packet drop events emitter with interval %s", option.Config.HubbleDropEventsInterval.String())
```

Comment:
Let's use structured logging here.
```suggestion
		logger.WithField("interval", option.Config.HubbleDropEventsInterval).Info("Starting packet drop events emitter")
```
Note: calling `.String()` is unnecessary here and can also be removed.

Reply:
Fixed, thanks.